cmd/operator/builder.go

```diff
@@ -178,7 +178,7 @@ func (builder CommandBuilder) runOperatorManager(kubeCfg *rest.Config, isDeploye
 		return err
 	}
 
-	otelShutdownFn := otel.Start(builder.getSignalHandler(), "dynatrace-operator", operatorManager.GetAPIReader(), builder.namespace)
+	otelShutdownFn := dtotel.Start(builder.getSignalHandler(), "dynatrace-operator", operatorManager.GetAPIReader(), builder.namespace)
```

Comment:
same as above:
> why are you not using the signalHandler?

Reply:
It is using `builder.getSignalHandler()`?!?